services/wallet/router/fees.go

```diff
@@ -132,47 +153,35 @@ func (f *FeeManager) SuggestedFees(ctx context.Context, chainID uint64) (*Sugges
 		return nil, err
 	}
 
-	fees, err := f.getFeeHistorySorted(chainID)
-	if err != nil {
-		return &SuggestedFees{
-			GasPrice:             weiToGwei(gasPrice),
-			BaseFee:              weiToGwei(baseFee),
-			MaxPriorityFeePerGas: weiToGwei(maxPriorityFeePerGas),
-			MaxFeePerGasLow:      weiToGwei(maxPriorityFeePerGas),
-			MaxFeePerGasMedium:   weiToGwei(maxPriorityFeePerGas),
-			MaxFeePerGasHigh:     weiToGwei(maxPriorityFeePerGas),
-			EIP1559Enabled:       false,
-		}, nil
-	}
-
-	perc10 := fees[int64(0.1*float64(len(fees)))-1]
-	perc20 := fees[int64(0.2*float64(len(fees)))-1]
-
-	var maxFeePerGasMedium *big.Int
-	if baseFee.Cmp(perc20) >= 0 {
-		maxFeePerGasMedium = baseFee
-	} else {
-		maxFeePerGasMedium = perc20
-	}
+	// Static for now until we develop more strategy
+	maxFees := new(big.Int).Add(new(big.Int).Mul(baseFee, big.NewInt(2)), maxPriorityFeePerGas)
 
-	if maxPriorityFeePerGas.Cmp(maxFeePerGasMedium) > 0 {
-		maxFeePerGasMedium = maxPriorityFeePerGas
-	}
+	return &SuggestedFees{
+		GasPrice:             gasPrice,
+		BaseFee:              baseFee,
+		MaxPriorityFeePerGas: maxPriorityFeePerGas,
+		MaxFees: &MaxFees{
```

Comment:
Is there a point in having this type, with Low/Medium/High values, when we're using the same value for all of them?

I am more for the variant to keep this type and set different values for Low/Medium/High (any logic is good) but then based on the input fee mode, select one of them. 
And client should determine that, for now it will be High mode, hardcodded on the client side.
Does that sound ok?